src/cljs/athens/electron/boot.cljs

```diff
@@ -74,13 +79,21 @@
                                                    [:navigate :home]
                                                    [:reset-undo-redo]
                                                    [:posthog/set-super-properties]
-                                                   [:loading/unset]]
-                                      ;; This event ends the async flow successfully.
-                                      :halt?      true}
+                                                   [:loading/unset]]}
+                                     {:when     :seen-all-of?
```

Comment:
Nit: this is a bit fragile, all it takes is for us to change one of these (or their flows) for the boot to fail mysteriously.